hexrd/ui/calibration/wppf_runner.py

```diff
@@ -82,9 +82,7 @@ def write_lattice_params_to_materials(self):
 
             mat.latticeParameters = lparms
             HexrdConfig().flag_overlay_updates_for_material(name)
-
-            if mat is HexrdConfig().active_material:
-                HexrdConfig().active_material_modified.emit()
+            HexrdConfig().material_modified(name)
```

Comment:
Should be `HexrdConfig().material_modified.emit(mat_name)`

Reply:
Fixed